app/views/external_users/claims/misc_fees/litigators/_misc_fee_fields.html.haml

```diff
@@ -1,22 +1,55 @@
+- fee = present(f.object)
+- claim = present(f.object.claim)
 .form-section.misc-fee-group.nested-fields.js-block.fx-do-init.fx-fee-group.fx-numberedList-item{ data: { 'type': 'miscFees', autovat: @claim.apply_vat? ? 'true' : 'false' } }
 
   = f.govuk_fieldset legend: { text: t('.misc_fee_html') } do
 
     = link_to_remove_association f, wrapper_class: 'misc-fee-group', class: 'govuk-link govuk-!-display-none fx-numberedList-action' do
       = t('common.remove_html', context: t('.misc_fee'))
 
-    = f.govuk_radio_buttons_fieldset(:fee_type_id, form_group: {class: 'fee-type'}, legend: { size: 's', text: t('.fee_type') }) do
-      - present_collection(@claim.eligible_misc_fee_types).each do |misc_fee_type|
-        = f.govuk_radio_button :fee_type_id,
-          misc_fee_type.id,
-          label: { text: misc_fee_type.description },
-          'data-unique-code': misc_fee_type.unique_code do
-          - if misc_fee_type.unique_code.eql?('MIUMO')
-            = render 'warnings/unused_material_over_3_hours'
-
-    = f.govuk_number_field :amount,
-      label: { text: t('.net_amount_html', context: t('.misc_fee')) },
-      class: 'total fee-rate',
+    = f.govuk_collection_select :fee_type_id,
+      claim.eligible_misc_fee_type_options_for_select.map{ |ft| [ft.first, ft.second, { 'data-unique-code': ft.third[:data][:unique_code] }] },
+      :second,
+      :first,
+      class: 'js-misc-fee-type js-fee-type js-fee-calculator-fee-type fx-misc-fee-calculation',
+      form_group: { class: 'fee-type fx-autocomplete-wrapper cc-fee-type' },
+      label: { text: t('.fee_type_html', context: t('.misc_fee')), size: 's' },
+      options: { include_blank: '' }
+
+    -# - if misc_fee_type.unique_code.eql?('MIUMO')
+    -#   = render 'warnings/unused_material_over_3_hours'
```

Comment:
Is this needed? Should it be commented out?

Reply:
This was fixed in a commit after the one you've commented on 🙂